pkg/agent/util/ndp/ndp.go

```diff
@@ -17,10 +17,16 @@ package ndp
 import (
 	"fmt"
 	"net"
+	"net/netip"
 
 	"github.com/mdlayher/ndp"
 )
 
+var (
+	// IPv6AllNodes is the IPv6 link-local multicast address for all nodes.
+	IPv6AllNodes = netip.MustParseAddr("ff02::1")
+)
```

Comment:
there is a function for this: https://pkg.go.dev/net/netip#IPv6LinkLocalAllNodes

Reply:
done